src/common/hooks/useOnboard.ts

```diff
@@ -39,16 +38,10 @@ export default function useOnboard() {
     if (initOnboard) {
       // These are optional callbacks to be passed into onboard.
       const subscriptions = {
-        address: (address: string | null) => {
-          dispatch({ type: actions.SET_ADDRESS, payload: address });
+        address: (addr: string | null) => {
```

Comment:
minuscole nit, I think this will never be `null`

Reply:
I think when it disconnects it might pass in null but I'm not 100% sure.